Mongo DB & Cassandra build fails [master]

Our Mongo DB builds in travis are failing with below

```
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/MongoBankAccountDTO.java:9: error: cannot find symbol
import javax.persistence.Lob;
                        ^
  symbol:   class Lob
  location: package javax.persistence
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:9: error: cannot find symbol
import javax.persistence.Lob;
                        ^
  symbol:   class Lob
  location: package javax.persistence
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/MongoBankAccountDTO.java:52: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.MongoBankAccountDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/MongoBankAccountDTO.java:57: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.MongoBankAccountDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:137: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:143: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:149: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:155: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:160: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:166: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:172: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:178: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:183: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:188: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:193: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:198: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
```

There are similar weird error for cassandra as well

```
[ERROR] /home/travis/app-cassandra/src/test/java/com/mycompany/myapp/web/rest/AccountResourceIntTest.java:[26,25] cannot find symbol
  symbol:   class Transactional
  location: package javax.transaction
[ERROR] /home/travis/app-cassandra/src/test/java/com/mycompany/myapp/web/rest/AccountResourceIntTest.java:[138,6] cannot find symbol
  symbol:   class Transactional
  location: class com.mycompany.myapp.web.rest.AccountResourceIntTest
[ERROR] /home/travis/app-cassandra/src/test/java/com/mycompany/myapp/web/rest/AccountResourceIntTest.java:[162,6] cannot find symbol
  symbol:   class Transactional
  location: class com.mycompany.myapp.web.rest.AccountResourceIntTest
[ERROR] /home/travis/app-cassandra/src/test/java/com/mycompany/myapp/web/rest/AccountResourceIntTest.java:[186,6] cannot find symbol
  symbol:   class Transactional
  location: class com.mycompany.myapp.web.rest.AccountResourceIntTest
[ERROR] /home/travis/app-cassandra/src/test/java/com/mycompany/myapp/web/rest/AccountResourceIntTest.java:[210,6] cannot find symbol
  symbol:   class Transactional
  location: class com.mycompany.myapp.web.rest.AccountResourceIntTest
[ERROR] /home/travis/app-cassandra/src/test/java/com/mycompany/myapp/web/rest/AccountResourceIntTest.java:[234,6] cannot find symbol
  symbol:   class Transactional
  location: class com.mycompany.myapp.web.rest.AccountResourceIntTest
[ERROR] /home/travis/app-cassandra/src/test/java/com/mycompany/myapp/web/rest/AccountResourceIntTest.java:[271,6] cannot find symbol
  symbol:   class Transactional
  location: class com.mycompany.myapp.web.rest.AccountResourceIntTest
[ERROR] /home/travis/app-cassandra/src/test/java/com/mycompany/myapp/web/rest/AccountResourceIntTest.java:[308,6] cannot find symbol
  symbol:   class Transactional
  location: class com.mycompany.myapp.web.rest.AccountResourceIntTest
[INFO] 8 errors 
```

seems like some travis environment issue @pascalgrimaud any idea?

--- a/generators/server/templates/_pom.xml
+++ b/generators/server/templates/_pom.xml
@@ -377,6 +377,10 @@
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-validator</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-envers</artifactId>
+        </dependency>
         <%_ if (databaseType == 'sql') { _%>
         <dependency>
             <groupId>org.hibernate</groupId>

